packages/common/src/lib/address.js

```diff
@@ -0,0 +1,49 @@
+/**
+ * Defines the shape of an address object
+ * @class
+ */
+class Address {
+	/**
+	 * @type {string} - the first line of the address
+	 */
+	addressLine1;
+
+	/**
+	 * @type {string} - the second line of the address
+	 */
+	addressLine2;
+
+	/**
+	 * @type {string} - the name of the town, city or other settlement
+	 */
+	town;
+
+	/**
+	 * @type {string} - the name of the county
+	 */
+	county;
+
+	/**
+	 * @type {string} - the postcode
+	 */
+	postcode;
+
+	/**
+	 * creates an instance of an address
+	 * @param {Object} params
+	 * @param {string} params.addressLine1
+	 * @param {string} params.addressLine2
+	 * @param {string} params.town
+	 * @param {string} params.county
+	 * @param {string} params.postcode
+	 */
+	constructor({ addressLine1, addressLine2, town, county, postcode }) {
+		this.addressLine1 = addressLine1;
+		this.addressLine2 = addressLine2;
+		this.town = town;
+		this.county = county;
```

Comment:
I think the other thing we talked about was that addressLine2 and postcode are optional so we could make them the last 2 constructor parameters and then also maybe thrown an error if addressLine1 and City / Town aren't provided (and maybe county if we need that?)